prereise/gather/Readme.txt

```diff
@@ -0,0 +1,88 @@
+

```

Comment:
I don't think this file should be included, it looks like a duplicate of `prereise/gather/hiflddata/Readme.txt`

Reply:
correct. that is some cleanup missed. will move it and double check the want and unwanted...